tests/unit/matrix/test_measure.py

```diff
@@ -1290,4 +1337,49 @@ def it_computes_its_blocks_to_help(self, request, dimensions_):
 
     @pytest.fixture
     def dimensions_(self, request):
-        return (instance_mock(request, Dimension), instance_mock(request, Dimension))
+        return instance_mock(request, Dimension), instance_mock(request, Dimension)
+
+
+class Describe_Zscores(object):
+    """Unit test suite for `cr.cube.matrix.measure._Zscores` object."""
+
+    def it_computes_zscore_subtotals_blocks_to_help(self, request, dimensions_):
+        weighted_cube_counts_ = instance_mock(request, _BaseWeightedCubeCounts)
+        _weighted_cube_counts = property_mock(
+            request, _BaseSecondOrderMeasure, "_weighted_cube_counts"
+        )
+        _weighted_cube_counts.return_value = weighted_cube_counts_
+        ZscoreSubtotals_ = class_mock(request, "cr.cube.matrix.measure.ZscoreSubtotals")
+        ZscoreSubtotals_.blocks.return_value = [[[1], [2]], [[3], [4]]]
+        zscores = _Zscores(dimensions_, None, None)
+
+        blocks = zscores.blocks
+
+        ZscoreSubtotals_.blocks.assert_called_once_with(
+            weighted_cube_counts_, dimensions_
+        )
+        assert blocks == [[[1], [2]], [[3], [4]]]
+
+    def it_computes_nan_subtotals_blocks_to_help(self, request, dimensions_):
+        weighted_cube_counts_ = instance_mock(
+            request, _BaseWeightedCubeCounts, zscores=[1, 2]
```

Comment:
maybe `zscores=[[1, 2], [3, 4]]` just so the reader doesn't have to puzzle over why these would be 1D.